tests/init-global/pos-tasty/simple.scala

```diff
@@ -0,0 +1,3 @@
+object O:
+  val a: Int = 5
+  val b = a.toDouble
```

Comment:
Should this test be able to work without Tasty?

Reply:
I've seen some cases where the call to `toDouble` is labeled as a Select tree, but this test case does not reproduce that

Reply:
This test case has been merged into `pos/arithmetic.scala`